flocker/provision/_install.py

```diff
@@ -505,7 +520,7 @@ def task_open_control_firewall(distribution):
     """
     if distribution in ('centos-7', 'fedora-20'):
         open_firewall = open_firewalld
-    elif distribution == 'ubuntu-14.04':
+    elif distribution in ('ubuntu-14.04', 'ubuntu-15.04'):
```

Comment:
No reason to revert them. It wouldn't surprise me if this makes ubuntu 15.04 just work. (But really, these should probably just check the distro portion, and not the version.)